src/geofabrics/lidar.py

```diff
@@ -31,7 +31,7 @@ def _load_lidar(self, lidar_file):
         land """
         
         pdal_pipeline_instructions = [
-            {"type":  "readers.las", "filename": lidar_file},
+            {"type":  "readers.las", "filename": str(lidar_file)},
```

Comment:
I think this is the best way, converting `lidar_file` to `str` rather than requiring a `str` be passed to the function :+1:

Reply:
@LukeParky Do you reckon I should type hint for str and pathlib.Path or type hint for pathlib.Path or leave it as type hint for str - as you can always ignore the type hint.

Reply:
I went with updating the type hint two to the two types I expect - thinking it over I think that sis clearer!

Reply:
Out of curiosity, does anything bad happen when you don't cast to str here but pass a Path?